src/common/body-scroll/index.js

```diff
@@ -0,0 +1,40 @@
+let previousPosition;
+let previousStyles;
+
+module.exports = {
+    /**
+     * Prevents the `<body>` element from scrolling.
```

Comment:
I think maybe just a little extra commenting, one or two lines, to explain how this is achieved.